### PR TITLE
save deploymentID to avoid mutating request entry in Audit

### PR DESCRIPTION
**Pull request by jiuker**
Do set when Use Rlocker

## Description

Do set when Use Rlocker

```go
if req.DeploymentID == "" {
   req.DeploymentID = xhttp.GlobalDeploymentID
}
```

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Comment by jiuker** [comment]
@calvernaz [pr 16054]https://github.com/minio/minio/pull/16054 which one is correct。

File: internal/logger/logger.go [hunk 1]
```diff
@@ -289,8 +289,9 @@ func errToEntry(ctx context.Context, err error, errKind ...interface{}) log.Entr
 
 	// Get the cause for the Error
 	message := fmt.Sprintf("%v (%T)", err, err)
+	DeploymentID := req.DeploymentID
```
Comment (klauspost) [inline internal/logger/logger.go]:
```suggestion
	deploymentID := req.DeploymentID
```

**Review by klauspost — CHANGES_REQUESTED** [review]
Yeah, I prefer this. Please make var lower case, though

**Review by klauspost — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**